Update index.html.ejs

jonathanmadeley opened the pull request:
Added MinecraftUUID.com.

jomo commented:
Hi, I'm happy to replace mcuuid.net with minecraftuuid.com in the "Try it" section, as it has no third party scripts / trackers. But please don't change the links in the side bar. Can you update the PR and squash your commits?

jonathanmadeley commented:
Hi there, apologies for the pull request - I was a tad confused with what
was acceptable. I've removed the link in the side bar and squashed it into
a single pull request.

Thanks again,

Jonathan.


On Sun, Oct 4, 2020 at 11:06 PM jomo <notifications@github.com> wrote:

> Hi, I'm happy to replace mcuuid.net with minecraftuuid.com in the "Try
> it" section, as it has no third party scripts / trackers. But please don't
> change the links in the side bar. Can you update the PR and squash your
> commits?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/crafatar/crafatar/pull/274#issuecomment-703322820>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACERQ7EJJCWR5EWJQQNFNH3SJDWXVANCNFSM4SAKGQ3A>
> .
>
